xknx/telegram/telegram.py

```diff
@@ -43,6 +45,7 @@ def __init__(
         self.direction = direction
         self.payload = payload
         self.source_address = source_address
+        self.reception_time = datetime.now()
```

Comment:
Telegrams are not only received, but sent too. I'd just call this `self.timestamp`